### ISSUE TITLE
[PSOBB] New Config Entries don't always correspond to their names-v4.json names

### ISSUE THREAD

**Issue by ShiftaDeband** [fuzziqersoftware/newserv#290]
This is on a Raspberry 5 8GB unit running the latest Raspberry Pi OS. I'm including that because I'm not sure if it makes a difference or not.

When adding the new config options for the 'QuestF960SuccessResultItems' and 'QuestF960FailureResultItems', I get the following error(s), which cascades to a few others.

```
terminate called after throwing an instance of 'std::runtime_error'
  what():  cannot parse item description "DB’s Saber" in BB_V4 (as text: item not found: db’s saber) (as data: invalid item class)
Aborted
```

This comes up on any entries with `’` in their text, and includes a few other items like `Earth Wand Brownie (Earth Wand: Brownie)` and `Ice Staff:Dagon (Ice Staff: Dagon)`.

Another issue was `Revival Cuirass` which is correct in the config, but the names-v4.json has this listed as `REVIVAL CURIASS`.

-- 

Long story short - I have a PR ready for this to go, but I'm not sure if this is the best approach to fix this. I also don't want to break anything for anyone where it's working and it's something on the OS side. Just trying to match the items to their names-v4.json names.

**Comment by fuzziqersoftware** [comment]
Thanks for fixing the names. I'll leave this open until I figure out why it didn't break for me on macOS - there's at least one other text-handling bug that happens on Linux only, so I'll spend some time investigating this soon.

**Comment by fuzziqersoftware** [comment]
This has been fixed by c1bcd45e - on my system, names like "DB's Saber" (with the smart quote character) were instead parsing as hex, giving the item code DBAE00, which is definitely wrong. The parser now fails if the first byte of the item data is 5 or higher, which should catch any instances of this in the future.
